Src/AutoFixture.xUnit.net2/AutoDataDiscoverer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    internal class AutoDataDiscoverer : DataDiscoverer
```

Comment:
You know what, I've thought about this further, and it seems unlikely to cause confusion or issues if we  expose it publicly, so I'll just make it public and avoid any potential issues.